### ISSUE TITLE
wish: helm-org-agenda-files-headings should indicate respective org-agenda file for each headline

### ISSUE THREAD

**Issue by GregorZattler** [emacs-helm/helm#902]
There might be serveral or even many org-agenda files with same headlines (e.g. one org-agenda file per project, every projekt with a headline "* Goal" or "* Budged" or whatever).  All these identical headlines are shown by helm-org-agenda-files-headings without any indication to which org-agenda file they belong.  It would be helpful to have the headings prefixed with the file name:
projecta: * Budged
projecta: * Goal
projecta: * Partners
project-b: * Budged
project-b: * Goal
project-b: * Issues
...
This is how headings are presented in the agenda itself.  For me it would be not important if the list was sorted by the headings or by the file names first.

Or to have non-selectable headings within the list of possible tagets:

= projecta =
* Budged
* Goal
* Partners
...

= project-b =
* Budged
* Goal
* Issues
...

Thanx for your attention, Gregor

**Comment by thierryvolpiatto** [comment]
Should not be too hard to fix, if someone wants to do it, he will have probably to truncate the line too.

**Comment by GregorZattler** [comment]
Hi Thierry,
* Thierry Volpiatto <notifications@github.com> [13. Mar. 2015]:
> Closed #902.

I realised your enhancement immediately while using helm
after an update: nice surprise.  Thanks.  

Ciao, Gregor
-- 
 -... --- .-. . -.. ..--.. ...-.-

**Comment by GregorZattler** [comment]
This worked great till Xmas 2015 now the file names are gone again.

With helm sources as of today I did a git bisect and this is the result:
db86d98 is the first bad commit
commit db86d98
Author: Thierry Volpiatto thierry.volpiatto@gmail.com
Date: Thu Dec 24 08:26:00 2015 +0100

Make more configurable helm org display.

* helm-org.el (helm-org-format-outline-path): New user var.
(helm-org-headings--nofilename): Removed.
(helm-org-show-filename): New user var.
(helm-org--get-candidates-in-file): Ensure window-with is called on helm-window (#1294).
Use new user vars.

:100644 100644 41a59406a20e8764e7d482d099f8321d78105076 6bbf80728fbfd63f0c2530eba02eb688b334a9aa M helm-org.el
0 ((db86d98...)|BISECTING)

Is it possible to re-enable/enhance helm to show file names in helm org headings ?

Thanks for looking into this, Gregor
